src/api/configurator.ts

```diff
@@ -46,7 +46,8 @@ export async function makeConfig(): Promise<IConfigFunction>
 				name: "crypto",
 				types: [...binancePairs],
 				providers: [
-					{ id: "binance", name: "Binance", types: binancePairs }
+					{ id: "binance", name: "Binance", types: binancePairs },
+					{ id: "bitfinex", name: "Bitfinex", types: binancePairs }
```

Comment:
`binancePairs` should be `bitfinexPairs`

Reply:
https://github.com/DucaturFw/ducor-backend/pull/17/commits/e1a34cd6dba4e282b855f0e60c2bd1e7a1a3ac93